selene/src/main.rs

```diff
@@ -517,17 +530,23 @@ fn start(mut options: opts::Options) {
                 }
             }
 
-            None => match fs::read_to_string("selene.toml") {
-                Ok(config_contents) => match toml::from_str(&config_contents) {
-                    Ok(config) => (config, None),
-                    Err(error) => {
-                        error!("Config file not in correct format: {}", error);
-                        std::process::exit(1);
-                    }
-                },
+            None => {
+                let read_config_file = |file: &str| -> Option<CheckerConfig<toml::value::Value>> {
+                    let config_contents = fs::read_to_string(file).ok()?;
+                    toml::from_str(&config_contents)
+                        .map_err(|error| {
+                            eprintln!("Error parsing config file {}: {}", file, error);
+                            error
+                        })
+                        .ok()
+                };
 
-                Err(_) => (CheckerConfig::default(), None),
-            },
+                let config = read_config_file("selene.toml")
+                    .or_else(|| read_config_file(".selene.toml"))

```

Comment:
Dotfiles should take precedence.

Reply:
fixed in https://github.com/Kampfkarren/selene/pull/591/commits/02213f435c44dfb46e66bb7e43a2aa52bd48ef9a